packages/react-dom/src/server/ReactDOMFizzServerNode.js

```diff
@@ -90,11 +90,26 @@ function renderToPipeableStream(
       hasStartedFlowing = true;
       startFlowing(request, destination);
       destination.on('drain', createDrainHandler(destination, request));
-      destination.on('close', createAbortHandler(request));
+      destination.on(
+        'error',
+        createAbortHandler(
+          request,
+          // eslint-disable-next-line react-internal/prod-error-codes
```

Comment:
Any particular reason this should not be minified?

Reply:
I’ll be honest, I’m not sure why any errors in fizz should be minified. The code stays in the server so payload is not the most critical thing and it forks the dev/prod behavior of onError which would likely lead to many confused devs not sure why their logic is broken in prod. And we don’t serialize the error and send to the client.

Is there a benefit to minifying that I’m missing?

Reply:
Most of the server builds (in particular the Node ones) don't minify but they do it globally for the whole build in `bundles.js`.

The browser ones do, partially because they're actually sometimes used in browsers. In particular renderToString is but you can also use streams e.g. in Service Workers.

That said, we should probably be size conscious even for the server because cold-starts on the server is such a significant issue.

So it should probably be up to those settings rather than on a per-error level.

Reply:
But what about the runtime divergence? How does one even test that their apps onError logic does what it is supposed to? Some errors are hard to trigger on purpose. I suppose this’ll be a challenge for onRecoverableError in the client. Perhaps a eager-lazy loading of the real error messages could be implemented if and only if you are trying to do logic with them?

Reply:
For logging, it's not really different than client-side logging is today. You log the minified error and then you can automatically reverse it from the codes.json in UI (like Sentry does I believe).

I imagine you typically would log them both to the same kind of system anyway.

Reply:
For things like branching on semantic logic like "if this was intentionally aborted, do something different" you really shouldn't use the error message anyway. E.g. built-in errors can be translated or change at the browser's will. There's things like `.name` or `instanceof` for that.

Reply:
As always, you're right :)